tests/install_hooks_test.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pytest
+import subprocess
+
+from git import Repo
+from rdm.cli import install_hooks
+
+
+@pytest.fixture
+def tmp_repo(tmpdir):
+    directory = str(tmpdir)
+    repo = Repo.init(directory)
+    os.chdir(directory)
+
+    yield repo
+
+    # teardown
+    subprocess.call(['rm', '-rf', directory])
+
+
+def test_install_hooks_no_destination(tmp_repo):
+    assert install_hooks(None) is None
```

Comment:
It seems like `install_hooks` always returns `None`.

See https://stackoverflow.com/questions/20274987/how-to-use-pytest-to-check-that-error-is-not-raised

I think the first suggestion in the answer is probably fine.

Including the asserts here is a bit misleading IMO